lib/Mojolicious/Command/openapi.pm

```diff
@@ -41,6 +41,7 @@ sub run {
     'p|parameter=s'          => \my %parameters,
     'c|content=s'            => \my $content,
     'S|response-size=i'      => sub { $ua{max_response_size} = $_[1] },
+    's|schema-version=s'     => \my $schema,
```

Comment:
Maybe this could just be "schema" instead of "schema-version"?

Reply:
Addressing this in the next push